src/app/dashboard/Drawer/Drawer.jsx

```diff
@@ -11,14 +11,16 @@ import { Link, useLocation } from "react-router-dom";
 
 import { Button } from "../../component";
 
+const drawerWidth = 216;
```

Comment:
Where did we come up with this number?

Reply:
This is from  `width: theme.spacing(18)` => 18 x 12 = 216. theme is outside of `useStyle` so I cannot use it here.

Reply:
Makestyles is a function, instead of the implicit return with makestyles => ({}) can you put the variable in
‘’’
Makestyles => {

Const drawerWidth ..

Return ({})
}
‘’’

?

If these elements really need this width, we should still reference the theme object as much as possible

Reply:
yeh as Josh said if you're just trying to resuse this value lets still use the `theme` object

```js
const useStyles = makeStyles(theme => {
const drawerWidth = theme.spacing(18);
return {...};
}
```

Reply:
Done, 👍